src/actions/repeatWithEach.ts

```diff
@@ -0,0 +1,49 @@
+import * as uuidv4 from 'uuid/v4';
+
+import WFWorkflowAction from '../interfaces/WF/WFWorkflowAction';
+import { withActionOutput } from '../utils';
+
+import WFSerialization from '../interfaces/WF/WFSerialization';
+/**
+ * @action RepeatWithEach
+ * @section Actions > Scripting > Control Flow
+ * @icon Scripting
+ *
+ * Takes a list of items as input, and runs the contained actions once for each item in the list.
+ *
+ * ```js
+ *  repeatWithEach({
+ *    actions: [showResult({})]
+ *  })
+ * ```
+ */
+const repeatWithEach = ({
+  items,
+  actions = [],
+}: {
+  /** The workflow actions inside the repeat block */
+  items?: WFSerialization;
```

Comment:
Not really sure, should we need to provide `input`? I found that other actions do not provide the input param but Workflow/shortcut will use the last action output, I am a little curious if we do not provide this, how do user pass variable cross actions between two or more action?